pixeldata/src/lib.rs

```diff
@@ -57,10 +57,13 @@
 
 use byteorder::{ByteOrder, NativeEndian};
 use dicom_core::{value::Value, DataDictionary};
+use dicom_encoding::adapters::DecodeError;
```

Comment:
These unused warnings can be fixed by adding `#[cfg(not(feature = "gdcm"))]` . But it might pollutes the imports too much and since we get rid of GDCM I didn't include it.